source/jormungandr/tests/ptref_tests.py

```diff
@@ -99,6 +99,19 @@ def test_line(self):
         geo = get_not_null(l, 'geojson')
         shape(geo)
 
+    def test_line_codes(self):
+        """test line formating"""
+        response = self.query_region("v1/lines/line:A?show_codes=true")
+
+        lines = get_not_null(response, 'lines')
+
+        assert len(lines) == 1
+
+        l = lines[0]
+
+        assert len(l["codes"]) == 3
```

Comment:
it's great to test that !

can you just add test on the content of the codes ? (+ some formating check in check_utils )